pallets/runtime/tests/src/committee_test.rs

```diff
@@ -96,7 +96,9 @@ fn prepare_proposal(ring: AccountKeyring) {
 }
 
 fn check_scheduled(id: PipId) {
-    assert_eq!(Pips::proposals(id).unwrap().state, ProposalState::Scheduled);
+    Pips::proposal_state(id).map(|pip_state| {
+        assert_eq!(pip_state, ProposalState::Scheduled);
+    });
```

Comment:
For tests it is ok to use `unwrap`.  The `map` wouldn't check for `None`, this change isn't valid.

To avoid the `unwrap`, you can wrap the expected value with `Some(<Expected state here>)`:
```suggestion
    assert_eq!(Pips::proposal_state(id), Some(ProposalState::Scheduled));
```